crowbar_framework/app/helpers/barclamp/suse_manager_client_helper.rb

```diff
@@ -25,7 +25,7 @@ def suse_manager_client_rpm
     end
 
     def suse_manager_client_install
-      "/opt/dell/bin/barclamp_install.rb --rpm suse-manager-client"
+      "knife cookbook upload core -o /opt/dell/chef/cookbooks"
```

Comment:
Nice, but still I do not see reall value in proposing it over simple knife cookbook upload. You need to run rake from specific directory for example.